pyfr/backends/cuda/compiler.py

```diff
@@ -93,7 +93,7 @@ def compile(self, name, src, flags=[]):
 
 
 class SourceModule(object):
-    def __init__(self, backend, src):
+    def __init__(self, backend, src, additional_opt=False):
```

Comment:
Still not sure about this; how about we instead expose flags to the ini file like we do with OpenMP?  Makes it nice and accessible for those who want to poke around as new features are added to NVRTC.